modules/ipromBidAdapter.js

```diff
@@ -0,0 +1,79 @@
+import * as utils from '../src/utils.js';
```

Comment:
can you please update this import to just pull in the needed functions. We recently went through all adapters and removed the '.utils' roots

Reply:
Done.